ultimatejobweb/API_FB.py

```diff
@@ -0,0 +1,29 @@
+# Import and load necessary lib
+import requests
+from bs4 import BeautifulSoup as bs
+
+class Manipulation:
+    def extract_jobs_list(self):
```

Comment:
I don't understand what this method is supposed to do

Reply:
Run over the soup object and return all URLs